README.md

```diff
@@ -14,8 +14,10 @@ It’s a React.js-based browser application with a Node.js backend with the foll
 
 ## Installation
 
-1. Ensure you have yarn ([installation instructions](https://yarnpkg.com/en/docs/install))
-2. Clone this repo
+1. Install Node v8.10.0
+    - We recommend [installing Node Version Manager][nvm-install], then: `nvm install 8.10.0`
+1. [Install][yarn-install] the `yarn` package manager.
+    - Note any specific instructions for `nvm` users.
```

Comment:
Not sure if I understand line `20`. Was it a note you left for yourself?

Reply:
Ah, got it. Makes sense. I like recommending `nvm` and the linux note. Thanks for this!